### ISSUE TITLE
Unable to build Alluxio tarball from scratch

### ISSUE THREAD

**Issue by LuQQiu** [Alluxio/alluxio#12764]
**Alluxio Version:**
I tested the below tags and branches, all unable to finish the `generate-tarballs single` successfully
tags: v2.4.0 v2.4.1 v2.4.1-1
branch: branch-2.4

**Describe the bug**
The error message is
```
[ERROR] Failed to execute goal on project alluxio-integration-fuse: Could not resolve dependencies for project org.alluxio:alluxio-integration-fuse:jar:2.4.0: Failed to collect dependencies at com.github.serceman:jnr-fuse:jar:0.5.3: Failed to read artifact descriptor for com.github.serceman:jnr-fuse:jar:0.5.3: Could not transfer artifact com.github.serceman:jnr-fuse:pom:0.5.3 from/to spring-releases (https://repo.spring.io/libs-release): Not authorized -> [Help 1]

[ERROR] Failed to collect dependencies at org.alluxio:alluxio-shaded-hadoop:jar:2.2.0: Failed to read artifact descriptor for org.alluxio:alluxio-shaded-hadoop:jar:2.2.0: Could not transfer artifact org.alluxio:alluxio-shaded-hadoop:pom:2.2.0 from/to spring-releases (https://repo.spring.io/libs-release): Authentication failed for https://repo.spring.io/libs-release/org/alluxio/alluxio-shaded-hadoop/2.2.0/alluxio-shaded-hadoop-2.2.0.pom 401 Unauthorized
```

**To Reproduce**
from a clean environment without package caches, clone alluxio repo, checkout the mentioned tags or branches, run 
```
$ dev/scripts/generate-tarballs single
```
the maven compile stage will give the mentioned error

**Expected behavior**
Expect successfully build the tarball

**Urgency**
Urgent, alluxio users not able to built tarball

**Additional context**
looks like repo.spring.io needs authentication now, and we are not able to use it as a package cache
https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020

Our community users report that this can be workaround by adding
```
<repositories>
    <repository>
        <id>central</id>
        <name>bintray</name>
        <url>https://jcenter.bintray.com</url>
    </repository>
</repositories>
```

**Comment by LuQQiu** [comment]
master tarball can be generated successfully

**Comment by LuQQiu** [comment]
fixed by https://github.com/Alluxio/alluxio/pull/12516

**Comment by LuQQiu** [comment]
Still confusing, branch-2.4 build failed but master succeed, they both have changes in  #12516

**Comment by apc999** [comment]
this issue is fixed in master, release-2.4.1-2, fixing branch-2.4 by https://github.com/Alluxio/alluxio/pull/12822
